crates/trycp_server/src/main.rs

```diff
@@ -0,0 +1,153 @@
+extern crate structopt;
+extern crate tempfile;
+
+//use log::error;
+//use std::process::exit;
+use self::tempfile::tempdir;
+use jsonrpc_core::{IoHandler, Params, Value};
+use jsonrpc_ws_server::ServerBuilder;
```

Comment:
This may be a lot easier to work with as a `jsonrpc-http-server` instead. There's no need for a persistent socket connection if we're not getting notifications back.

Reply:
I tried that at first, but it didn't work easily in my testing for some weird reason.  And then, I realized that we actually wanted this because we wanted the closing of the websocket to be the indicator of how to shut down the node right?

Reply:
Yeah I am OK with this